src/main/java/mat/client/cqlworkspace/InsertAttributeBuilderDialogBox.java

```diff
@@ -75,15 +76,16 @@
 	private static final String VALUE_SETS = "Value Sets";
 	private static final String CODES = "Codes";
 	private static final String NULLABLE = "Nullable";
+	private static final String FHIR = "FHIR";

```

Comment:
looks like it's not used...you use MeasureDetailsUtil.FHIR instead

Reply:
forgot to remove it.